README.md

```diff
@@ -102,16 +111,28 @@ logging_enabled: true
 logging_purge_confs: true
 ```
 
-2) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the local system and outputs into the local files.
+2) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the local system and outputs into the local files, which actions are predefined.
 ```
 logging_enabled: true
 logging_purge_confs: true
 logging_outputs:
   - name: local-files
     type: files
-    logs_collections:
-      - name: system-input
-        type: basics
+logging_inputs:
+  - name: system-input
+    type: basics
+logging_flows:
+  - name: flow0
+    inputs: [system-input]
+    outputs: [local-files]
+```
+The local files `type files` is a special output.  The predefined file type `logging_outputs` and specifying the logging_flows could be skippable.  So, the following configuration is the same as above.
```

Comment:
> The local files `type files` is a special output.

Do you mean that it is the default output, if no output is specified?

> The predefined file type `logging_outputs` and specifying the logging_flows could be skippable.

Do you mean that the default output are files such as `/var/log/messages`?  And the default flow is to write records from `system-input` to the default output?

Reply:
> > The local files `type files` is a special output.
> 
> Do you mean that it is the default output, if no output is specified?

Yes.

> > The predefined file type `logging_outputs` and specifying the logging_flows could be skippable.
> 
> Do you mean that the default output are files such as `/var/log/messages`? And the default flow is to write records from `system-input` to the default output?

Yes.  Correct.

Reply:
Then I think it would be clearer to say something like `If inputs are specified, but no flows or outputs are specified, the default is to write the input to the predefined system log files e.g. /var/log/messages.`

Reply:
Done.